app/controllers/notes_controller.ts

```
@@ -0,0 +1,59 @@
+import Note from '#models/note'
+import { saveNoteValidator } from '#validators/note'
+import type { HttpContext } from '@adonisjs/core/http'
+
+export default class NotesController {
+  async index({}: HttpContext) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)